testutils/vcamera/vcamera.go

```diff
@@ -0,0 +1,216 @@
+//go:build linux
+
+/*
+Package vcamera creates and streams video to virtual V4L2 capture devices on Linux.
```

Comment:
[q] should this also include the instructions of how to use the script we have in /etc/ to download all the necessary modprobe and loopback functionality ?

Reply:
Good call! done.